buildfix.sh

```diff
@@ -37,7 +37,11 @@ echo "Formatting WORKSPACE/BUILD files..."
 buildifier -r .
 
 echo "Building and running gofmt..."
-bazel run "${BAZEL_QUIET_FLAGS[@]}" //:gofmt -- -w .
+GO_SRCS=()
+while IFS= read -r line; do
+    GO_SRCS+=("$line")
+done <<< "$(git ls-files '*.go')"
```

Comment:
uber nit:

```suggestion
done < <(git ls-files '*.go')
```

this results in a pipe that streams the output into the while loop instead of buffering the full output into a string first.